app/views/initiatives/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="wrapper">
+  <h2><%= "#{@team_member.fullname}'s Initiative" %></h2>
+  <div class="card">
+    <h3 class="init-name"><%= @initiative.title %></h3>
+    <p class="init-description"><%= simple_format(@initiative.description) %></p>
+    <div>
+      <div class="init-footer w-row">
+        <div class="w-col w-col-2 padding-zero"><%= image_tag "icon-#{@initiative.completion ? 'completion': 'calendar'}.svg", class: "icon-target" %></div>
+        <div class="card-date w-col w-col-10">
+          <%= @initiative.target_date.to_s(:long_ordinal) unless @initiative.target_date.nil? %>
+        </div>
+      </div>
+      <hr>
+      <div>
+        <%= render @initiative.notes.order('created_at DESC') %>
```

Comment:
How can we move `order` out of the view?

Reply:
Resolved!